src/site-layout/SiteLayout.js

```diff
@@ -107,6 +108,7 @@ function SiteLayout({ tenantId }) {
       <FooterWrapper>
         <Footer />
       </FooterWrapper>
+      <InfoPanel />
```

Comment:
It's not entirely obvious to me how this component ultimately ends up receiving its `data` and `renderContents` props?